### DISCUSSION TITLE
PWA instalable application

### DISCUSSION THREAD

**Discussion by leostereo** [quasarframework/quasar#8905] [CLI - Capacitor / Cordova modes]
Hi guys , I would like to create a PWA application.
My concern is about IOS platform.
After browsing and installing quasar documentation loved pwa apps.
So have following question , 
How can I got this installation menu in my app ? 
Is it possible to have same result on IOS ? will try to get an iphone to test.
Any feedback about this would be wellcome.
Leandro.

**Comment by ontwikkelfabriek** [comment]
No you cannot have a inapp PWA install to device menu on IOS.

You have to do it like this:
https://www.idownloadblog.com/2018/07/19/howto-progressive-web-apps-iphone-ipad/
